docs/src/content/hardhat-runner/docs/advanced/scripts.md

```diff
@@ -2,7 +2,7 @@
 
 In this guide we will go through the steps of creating a script with Hardhat. For a general overview of using Hardhat refer to the [Getting started guide](../getting-started/index.md).
 
-You can write your own custom scripts that can use all of Hardhat's functionality. A classic use case is writing a deployment script for your smart contracts.
+You can write your own custom scripts that can use all of Hardhat's functionality. A classic use case is writing a script that prints the list of available accounts.
```

Comment:
Ignore that, I now read the rest of the updates to this document.

Maybe instead of calling it a "classic", we can do "A really simple example can be a script that prints the list of..."

Reply:
I opted for:

> You can write your own custom scripts that can use all of Hardhat's functionality; for example, in this guide, we will work through scripts that print the list of available accounts.